script.js

```diff
@@ -0,0 +1,96 @@
+const bgSource = './images/chess-img.png';
+const courses = [
+  {
+    logoSource: './images/codse.png',
+    title: 'CODSE',
+    subtitle: 'Certificación Certified Offensive and Defensive Security EXPERT',
```

Comment:
- [ ] kindly consider adding the small horizontal like under this subtitle to make this section look exactly as the one on the provided design check the circled lines on the  image below to understand it more 
- 
<img width="962" alt="Screen Shot 2023-04-21 at 20 11 41" src="https://user-images.githubusercontent.com/60146030/233706606-5937ae29-364b-4e45-8341-04f435a302cf.png">

Reply:
Thanks for your recommendation, It's done. 👍🏼